acceptance-tests/helpers/brokers/env.go

```diff
@@ -67,8 +68,9 @@ func (b Broker) releasedEnv() []apps.EnvVar {
 
 func (b Broker) latestEnv() []apps.EnvVar {
 	return []apps.EnvVar{
+		{Name: "GSB_COMPATIBILITY_ENABLE_BUILTIN_BROKERPAKS", Value: true},
```

Comment:
Is this needed, or was it left over from trying something out?

Reply:
It is needed to load the brokerpak. I haven't investigated if it makes sense to have the code that load the brokerpak depending on a feature flag in the Cloud Service Broker, but I will do it. That is the reason why there was no brokerpak when in the investigation that we made this morning